### PR TITLE
Added craft text to all forge items

### LINKED ISSUE #1216: HotM Level Requirement for applicable forge recipes
It would be really cool if you could add the HotM level requirement for all forge recipes that have a requirement (like [this one](https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/blob/master/items/REFINED_TUNGSTEN.json#L15-L23)).

This might be kind of weird though as some forge recipes require specific collections levels instead of or in combination with HotM level... Also things like Mole pet require fossils...

### PR DESCRIPTION
**Pull request by jani270**
Closes #1216

### CODE REVIEW

File: items/PERFECT_ONYX_GEM.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "§6§lLEGENDARY"
   ],
   "internalname": "PERFECT_ONYX_GEM",
-  "crafttext": "",
+  "crafttext": "Requries: Gemstone X",
```
Comment (MattTheCuber) [inline items/PERFECT_ONYX_GEM.json:17]: Doesn't this also require HotM 5 or something?

Comment (jani270) [reply items/PERFECT_ONYX_GEM.json:17]: > Doesn't this also require HotM 5 or something?

Wiki doesnt mention anything about a hotm req
also ingame: 
image

Comment (MattTheCuber) [reply items/PERFECT_ONYX_GEM.json:17]: image

Also Tungsten Plate, Umber Plate, and Glacite Amalgamation.

Comment (MattTheCuber) [reply items/PERFECT_ONYX_GEM.json:17]: https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/assets/32849887/8583c5a1-60d8-479b-88f2-64b0b77a3061

image

image image image

Comment (jani270) [reply items/PERFECT_ONYX_GEM.json:17]: I dislike them really hard.

Comment (jani270) [reply items/PERFECT_ONYX_GEM.json:17]: Will fix it tmr probably, im too tired for this.

Comment (jani270) [reply items/PERFECT_ONYX_GEM.json:17]: https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/commit/ac1a1a146c6a1cbcd5208e49384a39777a3bd11e

**Pull request merged**